WordPress/twentytwentyfive
# Try: Update the order of the section styles

**Description**
I updated the order of the section styles from light to dark.
I placed the files in a new folder called sections.

I did not change the names, they are still called Style 1-5.

**Screenshots**
![image](https://github.com/user-attachments/assets/87120076-943d-42a5-93d7-f90d82117620)

Before:
![image](https://github.com/user-attachments/assets/d4cdf761-726e-4016-8f4f-918fa63a6d2e)


**Testing Instructions**

Add a group or columns block and confirm that the style sections have the correct color, but a different name and order than before the PR.
Add a few patterns that have a style section, and confirm that the color is the same as before the PR.

Contact location and link -yellow
Link in bio heading, paragraph, links and full-height image -purple
Hero podcast -pink
Logos -off white/ beige
